### DISCUSSION TITLE
pyMOR Community Meeting 2023-04-26

### DISCUSSION THREAD

**Discussion by sdrave** [pymor/pymor#2012] [Announcements]
The next pyMOR Community Meeting will take place Wednesday, April 26, at 14:00 CEST and 8:00 EDT, online at:

link: https://bbb.mpi-magdeburg.mpg.de/meeting/pymor
password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:
- [x] release process (please check #1946 before)
- [x] pyMOR School 2023 organization (date will be fixed)
- [x] preparing for the 2023.1 release
- [x] [PyPI organizations](https://blog.pypi.org/posts/2023-04-23-introducing-pypi-organizations/)

Moderator: @sdrave

**Comment by lbalicki** [comment]
Minutes:

- Discussed Comments on Release process/schedule
- pyMOR School 2023 organization
  - Date is more or less set to: September 18 - 22 
- preparing for the 2023.1 release
  - Release Date: June 29th
  - Announcement should be made by release manager
  - Should discuss name inheritance in `project` at some later point
    - Schedule a separate meeting for this before and/or after next community meeting
  - we should probably remove demos from getting started -> instead have simple stationary and simple BT example
- PyPI organizations
  - it is an easy form PM might fill it out

Next Meeting: May 9th, 8am EST or 2pm CET
Moderator: Petar
